src/main/steps/sequence.ts

```diff
@@ -0,0 +1,172 @@
+import { Request } from 'express';
+
+import {
+  HAS_RELATIONSHIP_BROKEN_URL,
+  PARTNER_GENDER_URL,
+  RELATIONSHIP_NOT_BROKEN_URL,
+  UNION_CERTIFICATE_URL,
+} from './urls';
+
+export interface Step {
+  id: string;
+  title: string;
+  field: string;
+  url: string;
+  subSteps?: SubStep[];
+}
+
+interface SubStep extends Step {
+  when: (response: Record<string, string>) => boolean;
+  finalPage?: boolean;
+}
+interface IncompleteStep {
+  id?: string;
+  title: string;
+  field?: string;
+  url?: string;
+  subSteps?: IncompleteSubStep[];
+}
+
+interface IncompleteSubStep extends IncompleteStep {
+  when: (response: Record<string, string>) => boolean;
+  finalPage?: boolean;
+}
+
+const sequence: IncompleteStep[] = [
+  {
+    id: 'partner-gender',
+    title: 'Who are you divorcing?',
```

Comment:
why is the title here? shouldn't it just be in the content?

Reply:
I've moved the English title here to make it easier to understand the order of the questions and any of the substeps. The content will now receive the title from this array 🎉